Musakai.Blazor/Pages/Home.razor

```diff
@@ -1,15 +1,40 @@
 ﻿@using Musakai.Blazor.Components.Images.ImageSlider
 @page "/"
 
-<PageTitle>Home</PageTitle>
+<PageTitle>Daniel Klement</PageTitle>
 
-<ImageSlider Title="Daniel Klement" Description="C#/.NET vývojář a fotograf" Slides="_slides" />
+<section id="presentation">
+    <ImageSlider Title="Daniel Klement" Description="C#/.NET vývojář a fotograf" Slides="_slides" />
+</section>
+<section id="about-me">
+    <h2>O mně</h2>
+    <Image Src="your-photo.jpg" Alt="Vaše fotka" Title="" Width="50px" Height="50px" Loading="Enums.Loading.ImageLoading.Auto" SrcSet="" Sizes="" />
+
+    <p>Ahoj, jsem Daniel, vášnivý <strong>foitograf</strong> a <strong>C#/.NET vývojář</strong>...</p>
+
+    <div class="fun-facts">
+        <h3>Zajímavosti</h3>
```

Comment:
https://zapier.com/blog/elevator-pitch-example/

Check this out (and maybe google some more). Elevator pitch is a powerful concept you could use here.

Reply:
Sure thing, this is just a basic concept, not a final version, It's preparation of HTML. I'll be doing texts later, need more time to step out of the box.